index.html

```diff
@@ -0,0 +1,164 @@
+          <li><a href="" class="link">Join</a></li>
+          <li><a href="" class="link">Sponsor</a></li>
+          <li><a href="" class="link">News</a></li>
+        </ul>
+      </nav>
+      <h2 class="pitch1">Hello! Sharing World</h2>
+      <h1 class="heading">World Peace Summit 2022</h1>
+      <div class="message">
+        <p>
+          If you want peace, you don't talk to your friends. You talk to your
+          enemies.Imagine all the people living life in peace. You may say that
+          I'm a dreamer, but I'm not the only one.
+        </p>
+      </div>
+      <h3 class="date">2022.11.21(MON) ~ 12(TUE)</h3>
+      <p class="heading-2">
+        @WPS 2022 social inclusion and equity, New york November 2022
+      </p>
+    </section>
+
+    <!-- Main program section -->
+    <section class="main-section">
+      <h3>Main Program</h3>
+      <hr class="line" />
+      <section class="main-cards">
+        <div class="main-program">
+          <article class="program-card">
+            <img src="/img/lecture1.png" alt="" />
+            <h5>Guest</h5>
+            <p>
+              Listen to the speakers from various countries, about the messages
+              of peace and social inclusion.
+            </p>
+          </article>
+        </div>
+        <div class="main-program">
+          <article class="program-card">
+            <img src="/img/exhibiton.png" alt="" />
+            <h5>Documentary</h5>
+            <p>
+              Watch various documantary movies on peace and social inclusion
+              from differnt cultures around the world .
+            </p>
+          </article>
+        </div>
+        <div class="main-program">
+          <article class="program-card">
+            <img src="/img/forum.png" alt="" />
+            <h5>Interaction</h5>
+            <p>
+              Participate in discussions and give opinions on a long time
+              solution to end racial discrimination.
+            </p>
+          </article>
+        </div>
+        <div class="main-program">
+          <article class="program-card">
+            <img src="/img/workshop.png" alt="" />
+            <h5>Q&A session</h5>
+            <p>
+              Participate by asking questions to an honor guest of the day and
+              make a note of Answers.
+            </p>
+          </article>
+        </div>
+        <div class="main-program">
+          <article class="program-card">
+            <img src="/img/ignite.png" alt="" />
+            <h5>Connec & Create</h5>
+            <p>
+              Connect with others to make organizations with the aim of peace
+              conservation.
+            </p>
+          </article>
+        </div>
+      </section>
+      <button class="atc-btn">Join Our Community</button>
+      <a class="see-more" href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <!-- Featured speakers -->
+    <section class="speakers-section"></section>
```

Comment:
- [ ]  You have implemented the speakers' section in a good way. Great job 👍. Let's add the `more` button so your design matches the original design:
- Original design:
<img width="210" alt="image" src="https://user-images.githubusercontent.com/77190203/163313309-dd906196-5fd2-460f-b213-71121801a808.png">

Reply:
@medaminedev66 implementing the more button is optional in the requirements provided by microverse

Reply:
Hi @ndayishimiyeeric 👋🏿   the more button is part of the additional requirements, Good job implementing this functionality 🤗

Reply:
Hi @prolajumokeoni i have implemented the more button in the mobile version and it is working.